keyboards/1upkeyboards/1up60hte/keymaps/default/keymap.c

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2019 Bubnick
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "1up60hte.h"
```

Comment:
```suggestion
#include QMK_KEYBOARD_H
```
This will automatically include the correct header and is preferred for several reasons.